tests/optimizers/botorch_utils_test.py

```diff
@@ -4,6 +4,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import numpy as np
+import parameterspace as ps
```

Comment:
are you sure this import is needed? there is no new code that uses `ps`

Reply:
Thanks, will remove :relaxed: